### PR TITLE
Ignore virtual columns in insert statements

### PR DESCRIPTION
**Pull request by cmer**

### CODE REVIEW

File: lib/evil_seed/record_dumper.rb [hunk 1]
```diff
@@ -50,14 +50,21 @@ def transform_and_anonymize(attributes)
...
 
+    def insertable_column_names
+      model_class.columns_hash.reject { |k,v| v.virtual? }.keys
+    end
+
     def insert_statement
       connection = model_class.connection
       table_name = connection.quote_table_name(model_class.table_name)
-      columns    = model_class.column_names.map { |c| connection.quote_column_name(c) }.join(', ')
+      columns    = insertable_column_names.join(', ')
```
Comment (Envek) [inline lib/evil_seed/record_dumper.rb:60]: I believe it is better to keep quoting or it may fail on columns named the same as reserved SQL keywords.

**Review by Envek — COMMENTED** [review]
Thanks for the pull request!

There are some concerns about compatibility with other databases and Rails versions, but I will be happy to merge it after they resolved!

**Comment by cmer** [comment]
You're right on both counts. It's been fixed.

**Comment by Envek** [comment]
Released in 0.4.0, thanks!

**Pull request merged**